### ISSUE TITLE
System Wide - Fee Change Recalculation/ Budget String 

### ISSUE THREAD

**Issue by MLWilkins** [dhenry-KCI/FredCo-Post-Go-Live-#163]
the school construction fees that increased on 1/1/20 did not update on in life projects where the fee was previously populated and unpaid. This issue has been circulating through emails.  

From Dustin 1/2/20: Scott and Christian believe they have identified the issue with the school construction fees and are updating scripts to fix the data. The script will not be complete and after discussions, believe the script should be vetted with Dan before implementation.

**Comment by megisys** [comment]
Can I have a sample AP Number for my testing please ? Urgent

**Comment by MLWilkins** [comment]
13442
--
15662
15663
15889
15942
16102
16105
17307
17837
19484
19485
19486
19542
19543
19544
19545
19546
19913
19914
19915
19916
19917
19918
19919
19920

**Comment by MLWilkins** [comment]
AReed manually adjusted these fees due to the urgency, but we must insure that all fee types will auto update whenever there is a fee update.  Come July 1 we will have a huge number of incorrect fees if the system doesn't update fees.

**Comment by MLWilkins** [comment]
this needs to work for new IPS project as well as H& projects

**Comment by MLWilkins** [comment]
H7

**Comment by megisys** [comment]
OK We have too issues at play. Issue 1. We have another issue about Fee Mismatches being deployed soon to fix inlife fee issues. Issue 2. We didn't complete the Fee Recalculation piece . as I thought we had time before July 1 and this was touted as a post go live task.. Will work on this soon for July 1.

**Comment by megisys** [comment]
@MLWilkins  as this has been adjusted manually and the next fee change is not until July 1. Can you please mark as a lower priority as HIgh . High is fix within a day.

**Comment by MLWilkins** [comment]
Changes to low priority with understanding this broader issue of fees updating will be addressed well in advance of the July 1 2020 fee changes.

**Comment by megisys** [comment]
Need to build this and Allow for Training and Testing.

**Comment by MLWilkins** [comment]
changed to high priority.  This is a must have before funding runs out for this phase 1B

**Comment by megisys** [comment]
Funding availability should not change the priority of an issue. We have an active SLA in place and was agreed upon. Has the business impact change for now

**Comment by megisys** [comment]
There is now  a Batch Job for this. It needs to be run manually. But is in place.

![image](https://user-images.githubusercontent.com/47392306/73073029-6dfb1c80-3e84-11ea-89a8-9c195a4d6851.png)

**Comment by megisys** [comment]
Adding Test Accepted so it gets moved. But do not close this ticket.. Need to hand off to Jeff

**Comment by JuanitaMiles** [comment]
@megisys Spoke with the County and confirmed that we do not want to push this fix to production before it is thoroughly tested. Once we get past the initial implementation of the upgrade we need to identify the best strategy to test this batch job and then determine if we want to push to prod. Removing Test Accepted label.

I have Dustin assigned to coordinating a training session with Jeff and Infor. This will be part of that session.

**Comment by ABonomo21** [comment]
retitle- Planning-school construction fees on in life projects did not update 1/1/20

**Comment by ABonomo21** [comment]
Source:
Repository: dhenry-KCI / FredCo-Post-Go-Live-
Ticket #: 163
Requested By: Mike Wilkins
Jump to this repository

Request:
This issue was originally created to address the school construction fees that did not update on in-life projects upon the fee increase on 1/1/20. However it appears the issue is larger, as Infor will need to complete the fee recalculation configuration for when fee schedules are updated July 1 of each year, for the system as a whole.

Please see source issue above for more information.

**Comment by ABonomo21** [comment]
Source:
Repository: kcigeospatial / Fred_Co_Land-Management
Ticket #: 2294
Requested By: Ashlye Bonomo
Jump to this repository

Request:
Ticket initially created to correct impact fee budget codes, but then broadened to include ALL fees. Jeff Wisner worked on these and reported all fee budget codes should now be correct.

Need to confirm we have not identified any budget code issues post go-live.

**Comment by JuanitaMiles** [comment]
@megisys While this has not been fully tested and accepted by county staff, can we ensure this configuration that exists in TEST does not get wiped away during the refresh monday?

**Comment by megisys** [comment]
Added to Production..

**Comment by MLWilkins** [comment]
This has not been tested and is not ready to be closed

**Comment by megisys** [comment]
Tested with Jeff Wisner on the process and we tested updating permit fees and seeing the effect on open fees after running the batch job. Should not have been closed. But promoted to production so not lost during refresh.
